application/client.core/src/app/environment/components/tabs/plugins/details/component.ts

```diff
@@ -77,6 +77,9 @@ export class ViewPluginsDetailsComponent implements AfterContentInit, AfterViewI
     }
 
     public _ng_onInstallPlugin() {
+        if (this._ng_plugin.default) {
+            PluginsService.getManager().reinstallDefaultPlugin({ name: this._ng_plugin.name});
+        }
```

Comment:
I am not sure, the client's logic should depend on is plugin default or not. For client it's just a plugin. 
I believe this is the responsibility of backend (electron) to recognize the plugin and do some specific actions.